src/features/navigation/RootNavigator.tsx

```diff
@@ -41,7 +41,7 @@ export type RootStackParamList = {
   AppComponents: undefined
   CheatCodes: undefined
   ForgottenPassword: undefined
-  Login: undefined
+  Login: { shouldGoBackToHome: boolean }
```

Comment:
Un naming plus approprié à mon avis :
```suggestion
  Login: { shouldGoBackButtonRedirectToHome: boolean }
```